operator/pom.xml

```diff
@@ -260,6 +260,11 @@
             <artifactId>client-java</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.appscode.voyager</groupId>
+            <artifactId>client-java-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Two problems: 1) You can't depend on a SNAPSHOT build, and 2) you need to add a test scope element.

Reply:
Looks like they haven't published a release.  You will need to either not use this dependency or ask them to publish a release to maven.

Reply:
have created an issue to voyager for this: https://github.com/voyager-client/java/issues/5

Reply:
Looking into this.

Reply:
I have released `com.appscode.voyager:client-java:0.1.0`. https://oss.sonatype.org/content/groups/public/com/appscode/voyager/client-java/ . It should be usable now. It takes around 7-8 hours to show up in https://search.maven.org/#search%7Cga%7C1%7Cappscode

Reply:
I have updated the dependency to released version. It works fine.